lib/puppet_x/puppetlabs/strings/yard/handlers/host_class_handler.rb

```diff
@@ -16,7 +16,13 @@ class PuppetX::PuppetLabs::Strings::YARD::Handlers::HostClassHandler < PuppetX::
         param_type_info[pop_param.name] = Puppet::Pops::Types::TypeFactory.any()
       else
         begin
-          param_type_info[pop_param.name] =  tp.interpret_any(pop_param.type_expr)
+          # This is a bit of a hack because we were using a method that was previously
+          # API private. See PDOC-75 for more details
+          if Puppet.version.start_with?('4.4')
```

Comment:
It does not have to be bumped, it was changed just once. It should however not ned to compute it over and over again as the runtime will remain the same the entire run.
It would also be fine to use `tp.method(:interpret_any) == 2` as the predicate (which should also be cached).

it could be done like this:
```
if Puppet::Pops::Types::TypeParser.instance_method(:interpret_any).arity == 2
  def interpret(expr)
    tp.interpret_any(expr, nil)
  end
else
  def interpret(expr)
    tp.interpret_any(expr)
  end
end
```
i.e. conditionally add a method that is suitable for the puppet version. Then call the `interpret(expr)` method instead of calling the method on tp (on line 22 an 24) in this PR